main.go

```diff
@@ -10,15 +11,27 @@ import (
 	"strings"
 	"time"
 
-	"github.com/giantswarm/crd-docs-generator/service/git"
+	"github.com/spf13/cobra"
 
 	"github.com/Masterminds/sprig"
 	"github.com/ghodss/yaml"
+	"github.com/giantswarm/crd-docs-generator/service/git"
 	"github.com/giantswarm/microerror"
 	blackfriday "gopkg.in/russross/blackfriday.v2"
 	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 )
```

Comment:
The import from `github.com/giantswarm/crd-docs-generator`(internal package) should not be in the same group as external dependendcies. Also the empty line 15 should be removed.